test/e2e/walkthrough.go

```diff
@@ -0,0 +1,209 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	v1alpha1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1"
+	"github.com/kubernetes-incubator/service-catalog/test/e2e/framework"
+	"github.com/kubernetes-incubator/service-catalog/test/util"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/pkg/api/v1"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = framework.ServiceCatalogDescribe("walkthrough", func() {
+	f := framework.NewDefaultFramework("walkthrough-example")
+
+	upsbrokername := "ups-broker"
+
+	BeforeEach(func() {
+		//Provisioning user provider service broker
```

Comment:
Nit: this should be 'deploying' instead of 'provisioning', because provisioning has a very specific meaning in this project.